From 1317dbbb0f3711e27e38ff6df180c1799657a30f Mon Sep 17 00:00:00 2001
From: Tom Anderson <thomasanderson@chromium.org>
Date: Thu, 11 Sep 2025 10:17:14 -0700
Subject: [PATCH] [Wayland] Reenable WaylandWpColorManagerV1

This may be reenabled now that [1] has landed to prevent HDR rendering
when GPU acceleration is disabled.

[1] https://chromium-review.googlesource.com/c/chromium/src/+/6925760

Change-Id: Icf13fb1425edd4b1cc583aa2cc65053b818bc300
Fixed: 375959958
Enabled-by-default-reason: launching
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/6934777
Reviewed-by: Kramer Ge <fangzhoug@chromium.org>
Commit-Queue: Kramer Ge <fangzhoug@chromium.org>
Auto-Submit: Thomas Anderson <thomasanderson@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1514356}
---
 ui/ozone/platform/wayland/host/wayland_wp_color_manager.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/ozone/platform/wayland/host/wayland_wp_color_manager.cc b/ui/ozone/platform/wayland/host/wayland_wp_color_manager.cc
index 946669929c32a9..ef0d9f401860e4 100644
--- a/ui/ozone/platform/wayland/host/wayland_wp_color_manager.cc
+++ b/ui/ozone/platform/wayland/host/wayland_wp_color_manager.cc
@@ -19,7 +19,7 @@ constexpr uint32_t kMinVersion = 1;
 
 BASE_FEATURE(kWaylandWpColorManagerV1,
              "WaylandWpColorManagerV1",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 std::optional<wp_color_manager_v1_primaries> ColorSpaceToPrimaries(
     gfx::ColorSpace::PrimaryID primary_id) {
